cfy_manager/main.py

```diff
@@ -864,11 +876,17 @@ def remove(verbose=False, force=False, config_file=None):
 
     yum_remove(_get_packages())
 
+    installed_services = get_installed_services_names()
     if _are_components_installed():
-        _remove(INITIAL_INSTALL_FILE)
-
+        for installed_service in installed_services:
+            _remove(os.path.join(INITIAL_INSTALL_DIR, installed_service))
```

Comment:
I think this will interact badly with a three node cluster- if you run a remove on one component then it'll remove the installed flag file for all components?

Reply:
The installed services are taken from the config file. So basically if you are running `cfy_manager remove` without using the `-c` flag, it will delete the file of the last service that was installed. Which, if I understand everything correctly, should be fine.

Reply:
Ah, yes, good point, I missed that. Looks reasonable then.